diff --git a/arch/arm/boot/dts/amlogic/wetek_play.dtd b/arch/arm/boot/dts/amlogic/wetek_play.dtd
index 3fc5e4e..6ab5c3b 100755
--- a/arch/arm/boot/dts/amlogic/wetek_play.dtd
+++ b/arch/arm/boot/dts/amlogic/wetek_play.dtd
@@ -877,8 +877,8 @@ void root_func(){
         vend_data: vend_data{
             vendor_name = "Amlogic";           /* Max Chars: 8     */
             vendor_id = <0x000000>;                 /* Refer to http://standards.ieee.org/develop/regauth/oui/oui.txt   */
-            product_desc = "MX MBox G18Ref";        /* Max Chars: 16    */
-            cec_osd_string = "Amlogic MBox";        /* Max Chars: 14    */
+            product_desc = "WeTek Play";            /* Max Chars: 16    */
+            cec_osd_string = "WeTek Play";          /* Max Chars: 14    */
         };
         
 //$$ MATCH "HDMI_pwr-ctrl_match" = <&pwr_ctrl>
